packages/app/src/components/feature/screens/travel/TravelContinueButton/index.tsx

```
@@ -2,26 +2,31 @@ import SocketManager from "@service/socket/manager";
 import useTravelStateStore from "@store/travel";
 import { useLocalSearchParams } from "expo-router";
+import * as Location from "expo-location";
 
 const TravelContinueButton = () => {
   const { courseId } = useLocalSearchParams();
   const socketManager = SocketManager.getInstance();
   const { connectedURL } = useTravelStateStore();
 
-  const handleContinue = () => {
+  const handleContinue = async () => {
+    let { latitude, longitude } = (await Location.getCurrentPositionAsync({}))
```

ESLint

[error] 'latitude' is never reassigned. Use 'const' instead.

(prefer-const)